TestingApp/ViewController/ViewController.swift

```diff
@@ -406,7 +403,10 @@ extension ViewController {
 
     @IBAction private func toggleAuthentication() {
         catchingError {
-            try Glia.sharedInstance.configure(with: configuration) { [weak self] result in
+            try Glia.sharedInstance.configure(
+                with: configuration,
+                theme: Theme()
```

Comment:
Why does theme have to reconstructed here every time 'authenticate' is tapped?

Reply:
Because if I remove this parameter, Xcode picks the old configure method instead and deprecation warning appears.